.github/workflows/validate.yml

```diff
@@ -0,0 +1,68 @@
+name: validate
+on: [push, pull_request]
+jobs:
+  main:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Do we need windows here? May be worth trying but might drain free minutes since it doubles the number of runs.

Reply:
Unless I'm reading this incorrectly, there is no limit to how much time the jobs run in total, only in how long an individual job runs: https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration

There's also a limit on concurrency, but that's probably not a concern either I think.

Reply:
> There's also a limit on concurrency, but that's probably not a concern either I think.

There are only 20 per account. dom-testing-library alone now uses 8 instead of 4. I'm missing some context why we need to introduce the windows run.

Reply:
I guess I figured it was easy and "free," but I guess we can drop windows 👍